pkg/resource/stack/deployment.go

```diff
@@ -115,6 +115,11 @@ func SerializeDeployment(ctx context.Context, snap *deploy.Snapshot, showSecrets
 		enc = config.NewPanicCrypter()
 	}
 
+	var completeBulkEncrypt completeBulkOperation
+	if eventualSecretCrypter, ok := enc.(bulkSecretEncrypter); ok {
```

Comment:
Looks like the "eventual" terminology was dropped? Might be worth propagating that through in some of our naming choices if so.